ansible_collections/arista/avd/molecule/upgrade_v2.x_to_v3.0_evpn_underlay_isis_overlay_ibgp/intended/structured_configs/DC1-BL1A.yml

```diff
@@ -9,11 +9,11 @@ router_bgp:
     OVERLAY-PEERS:
       type: evpn
       update_source: Loopback0
-      remote_as: 65001
       bfd: true
       password: 64fqSH5CFUNLRHErezMrRg==
       send_community: all
       maximum_routes: 0
+      remote_as: 65001
```

Comment:
Still an integer here

Reply:
this comes from stuctured config. not sure where I should look for conversion?

Reply:
https://github.com/aristanetworks/ansible-avd/blob/devel/ansible_collections/arista/avd/molecule/upgrade_v2.x_to_v3.0_evpn_underlay_isis_overlay_ibgp/eos_designs_upgrade_2.x_to_3.0/DC1-BL1A.yml#L53